### PR TITLE
Fix bug: [Issue #76] Response message not taking into account quantity of items

### LINKED ISSUE #76: String response to users not taking into account quantity of items
Examples: 
-  ```list``` command returns "Listed 1 students" instead of "student".
-  ```find``` command returns "1 students listed!" instead of "student".

### PR DESCRIPTION
**Pull request by nweiyue**
Could only find the issue in ```find``` command.
```clear``` and ```list``` command use "all students" so I didn't make any changes to them.

### CODE REVIEW

File: src/main/java/seedu/address/commons/core/Messages.java [hunk 1]
```diff
@@ -8,7 +8,22 @@
...
+     *
+     * @param numberOfStudents Number of students in the list.
+     * @return message to be displayed to user.
+     */
+
+    public static String getStudentListedMessage(int numberOfStudents) {
+        if (numberOfStudents == 1) {
+            return MESSAGE_SINGLE_STUDENT_LISTED_OVERVIEW;
+        }
+        return MESSAGE_MULTIPLE_STUDENTS_LISTED_OVERVIEW;
```
Comment (erisjacey) [inline src/main/java/seedu/address/commons/core/Messages.java]: Would this make more sense to put in an `if-else` block (with comments to describe the `else` block)?

**Review by erisjacey — APPROVED** [review]
LGTM! Just one minor nitpick.

**Review by marcustw — COMMENTED** [review]
LGTM

File: src/main/java/seedu/address/commons/core/Messages.java (hunk 1 above)
Comment (nweiyue) [reply src/main/java/seedu/address/commons/core/Messages.java]: Right. Thanks for pointing it out!

**Pull request merged**